modules/builds-use-custom-builder-image.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * assembly/builds
+// * builds/custom-builds-buildah.adoc
+
+
+[id='builds-use-custom-builder-image-{context}']
+= Use custom builder image
+
+You can define a BuildConfig that uses the custom strategy in conjunction with
+your custom builder image to execute your custom build logic.
+
+.Prerequisites
+
+* Define all the inputs that will go into creating your new custom builder image.
+* Build your custom builder image.
+
+.Procedure
+
+. Create a file named *buildconfig.yaml*. This file defines the BuildConfig that
+is created in your project and executed:
```

Comment:
I'd replace "execute." From the IBM Style Guide:
> ! execute v.
> Use only in technical documentation and only if a simpler, less technical term, such as “run” or “issue,” or a
more specific term, such as “start” or “enter,” is inappropriate. If “execute” is used as a keyword, you can
use it in related text to maintain consistency.